test/fakefs_helper.rb

```diff
@@ -24,3 +24,14 @@ def home(user = Etc.getlogin)
   end
 end
 FakeFS::Dir.singleton_class.prepend(FakeFSPatch)
+
+# Patch FakeFS::Pathname to include the path method for Ruby 4.0.0 compatibility
+# This is needed because Reek's configuration internally calls Pathname#== which
+# requires the path method to be present
+module FakeFS
+  class Pathname
+    def path
```

Comment:
```suggestion
    undef :path

    def path
```

Reply:
That's to get rid of the `warning: method redefined; discarding old path` at the start of tests.